pkg/kn/commands/service_delete.go

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2018 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewServiceDeleteCommand represent 'service delete' command
+func NewServiceDeleteCommand(p *KnParams) *cobra.Command {
+
+	serviceDeleteCommand := &cobra.Command{
+		Use:     "delete <SERVICE_NAME>",
+		Aliases: []string{"del", "d"},
```

Comment:
let's remove aliases for now. we should probably come up with consistent rules where to apply them.

Reply:
:+1: Logged the issue for same.